tests/capitalize.test.js

```diff
@@ -0,0 +1,5 @@
+const capitalize = require('../src/capitalize');
+
+test('capitalizes the first character in the string', () => {
```

Comment:
- [ ] Add another test case for this method. You should have at least 2 test cases, otherwise you can't tell if it's just passing the first by a fluke or hardcode.